### PR TITLE
Some quick refacto to command documentation & add doc for scan --quiet

### PR DESCRIPTION
**Pull request by moadibfr**

### CODE REVIEW

File: docs/usage/cmd/scan/usage.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+id: scan-usage
+title: Usage
+---
+
+```shell
+$ driftctl scan [OPTIONS]
+$ driftctl scan --from tfstate+s3://terraform.tfstate  --to github+aws --output console://
```
Comment (sundowndev) [inline docs/usage/cmd/scan/usage.mdx:8]: >unsupported cloud provider 'github+aws'
>Valid values are: aws+tf,github+tf

Not sure if it's a typo or not, what would github+aws be about ?

File: docs/usage/cmd/scan/usage.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+```
+
+## Description
+
+Scan resources from the input terraform statefile and compare it to your current profile infrastructure
```
Comment (sundowndev) [inline docs/usage/cmd/scan/usage.mdx:13]: Terraform*

```diff
-profile infrastructure
+profile infrastructure.
```

**Review by eliecharra — APPROVED** [review]

File: docs/usage/cmd/scan.mdx [hunk 3]
```diff
@@ -131,4 +223,31 @@ Found 1 resource(s)
  - 0 deleted on cloud provider
  - 1/1 drifted from IaC
 You have diffs on computed fields, check the documentation for potential false positive drifts
-```
\ No newline at end of file
+```
+
+## --to
+
+driftctl supports multiple provider. By default it will scan against aws but you can change this using --to.
```
Comment (wbeuil) [inline docs/usage/cmd/scan.mdx:230]: `AWS`

File: docs/usage/cmd/scan.mdx [hunk 4]
```diff
@@ -131,4 +223,31 @@ Found 1 resource(s)
...
+
+### Usage
+
+Environment: `DCTL_TO`
+
+```shell
+$ driftctl scan --to PROVIDER+TYPE
+
+# examples :
```
Comment (wbeuil) [inline docs/usage/cmd/scan.mdx:239]: `examples:`

File: docs/usage/cmd/scan.mdx [hunk 5]
```diff
@@ -131,4 +223,31 @@ Found 1 resource(s)
...
+$ driftctl scan --to aws+tf
+$ DCTL_TO=to:github+tf driftctl scan
```
Comment (wbeuil) [inline docs/usage/cmd/scan.mdx:241]: `DCTL_TO=github+tf`

File: docs/usage/cmd/scan.mdx [hunk 6]
```diff
@@ -131,4 +223,31 @@ Found 1 resource(s)
...
+```
+
+### Supported Providers
+
+driftctl support these provider:
```
Comment (wbeuil) [inline docs/usage/cmd/scan.mdx:246]: `supports these providers:`

File: docs/usage/cmd/scan.mdx [hunk 7]
```diff
@@ -1,23 +1,115 @@
...
 ---
 
+```shell
+$ driftctl scan [OPTIONS]
+$ driftctl scan --from tfstate+s3://terraform.tfstate  --to github+aws --output console://
+```
+
+## Description
+
+Scan resources from the input terraform statefile and compare it to your current profile infrastructure
```
Comment (wbeuil) [inline docs/usage/cmd/scan.mdx:13]: `infrastructure.`

Comment (wbeuil) [reply docs/usage/cmd/scan.mdx:13]: `Terraform`

File: docs/usage/cmd/scan.mdx (hunk 3 above)
Comment (wbeuil) [reply docs/usage/cmd/scan.mdx:230]: `multiple providers.`

**Review by wbeuil — CHANGES_REQUESTED** [review]
Why the package-lock has been deleted ?

**Review by wbeuil — APPROVED** [review]

**Pull request merged**